operator/templates/stateful-set.yaml

```diff
@@ -196,50 +256,90 @@ spec:
                   fieldPath: metadata.uid
           resources:
             requests:
-              memory: "{{ .Params.PROMETHEUS_EXPORTER_MEM_MIB }}Mi"
-              cpu: "{{ .Params.PROMETHEUS_EXPORTER_CPU_MC }}m"
+              memory: "{{ $.Params.PROMETHEUS_EXPORTER_MEM_MIB }}Mi"
+              cpu: "{{ $.Params.PROMETHEUS_EXPORTER_CPU_MC }}m"
             limits:
-              memory: "{{ .Params.PROMETHEUS_EXPORTER_MEM_LIMIT_MIB }}Mi"
-              cpu: "{{ .Params.PROMETHEUS_EXPORTER_CPU_LIMIT_MC }}m"
+              memory: "{{ $.Params.PROMETHEUS_EXPORTER_MEM_LIMIT_MIB }}Mi"
+              cpu: "{{ $.Params.PROMETHEUS_EXPORTER_CPU_LIMIT_MC }}m"
           volumeMounts:
             - name: cassandra-exporter-config-yml
               mountPath: /etc/cassandra_exporter/config.yml
               subPath: config.yml
         {{ end }}
+      {{ if $.Params.NODE_TOPOLOGY }}
+      initContainers:
+        - name: node-resolver
+          image: bitnami/kubectl:latest
```

Comment:
Using `latest` could be problematic, as this might differ from the version of the cluster. In the worst case this could cause issues. We should either use a tag that matches the min-required version of Kubernetes as stated in `operator.yaml` or make the tag a parameter.

Reply:
Good point. I'm going to change that